auv_description/auv_common_descriptions/urdf/sensors/bar30.urdf.xacro

```diff
@@ -19,18 +19,19 @@
         </geometry>
       </collision>
       <inertial>
-        <mass value="0.5" />
-        <inertia ixx="0.008545" ixy="0.0" ixz="0.0" iyy="0.008545" iyz="0.0" izz="0.0144" />
+        <mass value="0.00001" />
+        <inertia ixx="0.000001" ixy="0.0" ixz="0.0" iyy="0.000001" iyz="0.0" izz="0.000001" />
       </inertial>
     </link>
 
     <link name="${name}_link" />
 
-    <joint name="${name}_joint" type="fixed">
+    <joint name="${name}_joint" type="revolute">
```

Comment:
This needs to be fixed back in, you may add additional frame and a joint which is revolute.